src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,7 +9,8 @@ public class CliSyntax {
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
-    public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_GITHUB = new Prefix("g/");
+    public static final Prefix PREFIX_TELEGRAM = new Prefix("t/");
+    public static final Prefix PREFIX_TAG = new Prefix("tg/");
```

Comment:
Should we just set the `PREFIX_TAG` to `tag/` instead? 
`tg/` might be confused with telegram